### PR TITLE
Add extra space and text trimming for static viz axes labels

### PR DESCRIPTION
**Pull request by ranquild**
Adds extra space for axes based on the length of the text:
- X when labels are rotated by 45 degrees
- Y in all cases

Adds text trimming for bar charts for X labels only (I expect it to be the most common issue).

How to test via internal page:
- Go to `/_internal/static-viz`
- Change data in `StaticVizPage` to test different cases, e.g. more labels, longer labels

Extra space:
Screen Shot 2021-09-14 at 12 12 13 pm
Screen Shot 2021-09-14 at 12 20 25 pm
Screen Shot 2021-09-14 at 12 20 37 pm
Screen Shot 2021-09-14 at 12 24 45 pm

Text trimming:
Screen Shot 2021-09-14 at 4 48 30 pm
Screen Shot 2021-09-14 at 4 49 23 pm

### CODE REVIEW

File: frontend/src/metabase/internal/pages/StaticVizPage.jsx [hunk 1]
```diff
@@ -94,14 +94,27 @@ export default function StaticVizPage() {
           <StaticChart
             type="categorical/bar"
             options={{
-              data: [["Gadget", 20], ["Widget", 31]],
+              data: [
+                ["Alden Sparks", 70],
```
Comment (ranquild) [inline frontend/src/metabase/internal/pages/StaticVizPage.jsx:98]: Random generated names

**Comment by ranquild** [comment]
@metabase-bot run visual tests

**Review by dpsutton — APPROVED** [review]

**Comment by ranquild** [comment]
@metabase-bot run visual tests

**Review by kdoh — APPROVED** [review]
Looks good! Nice work.

**Pull request merged**